chromium_src/components/update_client/request_sender.cc

```diff
@@ -33,8 +33,12 @@ class BraveEcdsa : public Ecdsa {
  public:
   static std::unique_ptr<Ecdsa> Create(int key_version,
                                        const std::string_view& public_key) {
-    std::string base64_decoded = GetKey(kBraveKeyPubBytesBase64);
-    return Ecdsa::Create(kBraveKeyVersion, base64_decoded);
+    if (brave::ShouldUseOmaha4()) {
+      std::string base64_decoded = GetKey(kBraveKeyPubBytesBase64);
```

Comment:
```suggestion
      const std::string base64_decoded = GetKey(kBraveKeyPubBytesBase64);
```

Reply:
Applied in e88e0817b8e2fc4dc97b870db1c635f7b7c74433.